src/main/java/stream/flarebot/flarebot/commands/music/SkipCommand.java

```diff
@@ -53,120 +62,99 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                 MessageUtils.sendErrorMessage("You cannot skip if you aren't listening to it!", channel);
                 return;
             }
-            if (votes.containsKey(channel.getGuild().getId())) {
-                String yes = String.valueOf(votes.get(channel.getGuild().getId()).values().stream()
-                        .filter(vote -> vote == Vote.YES)
-                        .count());
-                String no = String
-                        .valueOf(votes.get(channel.getGuild().getId()).size() - Long.valueOf(yes));
-                channel.sendMessage(MessageUtils.getEmbed(sender).setColor(new Color(229, 45, 39))
-                        .setDescription("Can't start a vote right now! " +
-                                "Another one in progress! Please use `" + guild.getPrefix() + "skip YES|NO` to vote!")
-                        .addField("Votes for YES:", yes, true)
-                        .addField("Votes for NO:", no, true).build()).queue();
-            } else getVotes(channel, member);
+            if (VoteUtil.contains(skipUUID, guild.getGuild()))
+                MessageUtils.sendWarningMessage("Their is already a vote to skip current song! Vote with `{%}skip yes | no`", channel, sender);
+            else {
+                VoteGroup group = new VoteGroup("Skip current song", skipUUID);
+                List<User> users = new ArrayList<>();
+                for(Member inChannelMember : channel.getGuild().getSelfMember().getVoiceState().getChannel().getMembers()) {
+                    if(channel.getGuild().getSelfMember().getUser().getIdLong() != inChannelMember.getUser().getIdLong()) {
+                        users.add(inChannelMember.getUser());
+                    }
+                }
+                group.limitUsers(users);
```

Comment:
Formatting of lines above, also add a limitUsers(Collection)
And do addAll(collection) in the body.
Then just replace this with group.limitUsers(getVoiceState().getChannel().getMembers())

Reply:
The formatting is fine?

Reply:
Not really. Ctrl+Alt+l I believe to format. Use that.

Reply:
I've been formatting the entire time